src/git_changelog/commit.py

```diff
@@ -80,10 +88,12 @@ def __init__(
 
         tag = ""
         for ref in refs.split(","):
-            ref = ref.strip()  # noqa: PLW2901
-            if ref.startswith("tag: "):
-                tag = ref.replace("tag: ", "")
-                break
+            tag_ref = ref.strip()
+            if tag_ref.startswith("tag: "):
+                tag_ref = tag_ref.replace("tag: ", "")
+                if _is_semver(tag_ref):
+                    tag = tag_ref
+                    break
```

Comment:
Any reason why you use a new `tag_ref` variable instead of updating `ref`, apart from removing the need for the noqa comment?

Reply:
No reason other than avoiding another noqa comment. Do you prefer reusing the `ref` variable? Feel free to change it (or tell me to do it :-))

Reply:
Yeah I find reusing `ref` to be more readable, compared to having `ref`, `tag_ref` and `tag`. So I'd appreciate if you can update it :)

Reply:
Done